docs/content/concepts/assets/asset-checks.mdx

```diff
@@ -148,6 +148,67 @@ defs = Definitions(
 
 There are a variety of types supported via the <PyObject object="MetadataValue" /> class. You can view the metadata on the Checks tab of the asset detail page.
 
-## Executing checks via the UI
+## Executing checks
+
+### Via the UI
```

Comment:
My comments from the other PR apply here - the **Executing checks** section needs a horizontal rule before it, and there are some formatting things in this section that'll need to be corrected. If this PR is stacked on the other one, we should be good here.